### PR TITLE
Replace rdmd dependency with single dub package

### PR DESCRIPTION
**Pull request by andre2007**
`import std;` is not used by purpose to not cause issues for older D versions.

### CODE REVIEW

File: tls/dub.sdl [hunk 1]
```diff
@@ -26,44 +26,60 @@ configuration "openssl" {
...
+	import std.process;
+	import std.stdio;
+	import std.string;
+	import std.range;
+	import std.uni;
+	import std.file;
+	import std.conv;
+	import std.algorithm;
+	import std.path;
+    import std.functional;
```
Comment (Geod24) [inline tls/dub.sdl:42]: Indent is off. Also can you order those ?

**Comment by Geod24** [comment]
Better seen with whitespace diff off. LGTM minus a style nit.

File: tls/dub.sdl (hunk 1 above)
Comment (andre2007) [reply tls/dub.sdl]: Fixed

**Comment by andre2007** [comment]
Is there currently an issue with AppVeyor? It seems it crashes on linking a test on windows x86 environments.

**Pull request merged**